app/src/main/java/com/preonboarding/sensordashboard/common/base/BaseFragment.kt

```diff
@@ -0,0 +1,35 @@
+package com.preonboarding.sensordashboard.common.base
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.LayoutRes
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import androidx.fragment.app.Fragment
+
+abstract class BaseFragment<T : ViewDataBinding>(@LayoutRes val layoutRes: Int) : Fragment() {
```

Comment:
```suggestion
abstract class BaseFragment<VB : ViewDataBinding>(
    @LayoutRes private val layoutId: Int
) : Fragment(layoutId) {
    private var _binding: VB? = null
    protected val binding: VB
        get() = requireNotNull(_binding)

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        _binding = DataBindingUtil.bind(view)
        with(binding) {
            lifecycleOwner = viewLifecycleOwner
        }
    }

    override fun onDestroyView() {
        super.onDestroyView()
        _binding = null
    }
}
```
요건 제가 쓰는 베이스Fragment인데 Fragment() 가 아닌 Fragment(layoutRes)으로 하면 inflate가 아닌 DataBindingUtil.bind(view)로 코드 몇줄 줄일 수 있습니다. 참고만 해주세요!
![image](https://user-images.githubusercontent.com/86879099/192264125-edf92f61-4ddc-49b1-ad93-a80d9356937a.png)